src/themes/teams/utils/index.ts

```diff
@@ -1,7 +1,22 @@
-import { ThemeInput } from '../../types'
+import { ThemeInput, ColorScheme, ColorSchemeMapping } from '../../types'
+import { Partial } from '../../../../types/utils'
 
 export const getSideArrow = (theme: ThemeInput) => {
   const { rtl, siteVariables } = theme
   const { arrowLeft, arrowRight } = siteVariables
   return rtl ? arrowLeft : arrowRight
 }
+
+export const getColorSchemeWithCustomDefaults = (
+  customDefaultValues: Partial<ColorScheme>,
+  colorScheme: ColorSchemeMapping,
```

Comment:
nit: I'd make `colorScheme` the first argument

Reply:
Will address this, and also move this to colorUtils